Tests/Integration/DSCResources/Adobe.config.ps1

```diff
@@ -25,53 +25,34 @@ configuration Adobe_config
         $SkipRuleType,
 
         [Parameter()]
-        [hashtable]
-        $Exception,
+        [string[]]
+        $SkipRuleCategory,
```

Comment:
Is SkipRuleCategory the right name for this parameter?  In the stigData it's called severity as well as in the STIG.  Does the parameter name SkipRuleSeverity make more sense?  It would keep the terminology consistent.  We could probably use a ValidateSet here also (High, medium, low, or CAT1, CAT2, CAT3, etc...)

Reply:
So I contemplated this actually.  I finally decided to implement (CAT I, II and III) what was commonly referred to in both the xccdf, when the xsl style sheet is used, as well as what STIGViewer displays as well. However, I do agree that the parameter name may not be the best, it may make sense since both the STIG displayed via the style sheet and STIGViewer reference "Severity", to rename the parameter "SkipRuleSeverity".  I would disagree however that allowing both values "high and CAT I" is better, as more code would be used as well as the potential for customer confusion.

Reply:
I agree with Jason on the parameter name and parameter set.